packages/components/src/QueryModel/ChartMenu.tsx

```diff
@@ -16,16 +16,7 @@ interface Props extends RequiresModelAndActions {
     showSampleComparisonReports: boolean;
 }
 
-interface State {
-    selectedChart?: DataViewInfo;
-}
-
-export class ChartMenu extends PureComponent<Props, State> {
-    constructor(props) {
-        super(props);
-
-        this.state = { selectedChart: undefined };
-    }
+export class ChartMenu extends PureComponent<Props> {
     componentDidMount(): void {
         const { model, actions, showSampleComparisonReports } = this.props;
         actions.loadCharts(model.id, showSampleComparisonReports);
```

Comment:
nit: This is a bit outside the scope of this review but if the `ChartMenu` is going to be responsible for calling `actions.loadCharts` then it should also be responsible for ensuring updates result in the charts being reloaded. I know that the unmounting strategy is what is currently working the the "g" pages but I think it will lead to confusion when charts don't reload down the line.